playbook/L3/roles/l3-basic-functionality/tasks/l3-basic-functionality.yml

```diff
@@ -216,7 +215,7 @@
     command: bash -c "opx-show-route | grep {{route0}}"
     register: result
     failed_when: result.rc == 0   
-  tags: ['Verify user can add modify and delete ipv4 address to a vlan_interface','sanity']
```

Comment:
Removing sanity from this test case due to the following error:
oot@OPX:~# opx-show-vlan --id 100 <-- error seen in output (in current release)
Traceback (most recent call last):
File "/usr/bin/opx-show-vlan", line 199, in 
obj.show_vlans(name)
File "/usr/bin/opx-show-vlan", line 128, in show_vlans
self.print_vlan(obj[0])
IndexError: list index out of range

Expected output: _opx-show-vlan: No VLAN information found_